README.md

```diff
@@ -1,14 +1,21 @@
 # F# project
 
 Adding 7 to the number taken as input from the user via powershell
-# Commands
-  --dotnet build
-  --dotnet publish
-  --import the module
-  --run command "Add-Seven Input number" 
+## How to run
+--create new dotnet solution\
+--add f# library in it\
+--add  package PowerShellStandard.Library\
+--copy the code from src/library.fs\
+--add script from scripts folder
```

Comment:
```suggestion
--create new dotnet solution \
--add f# library in it \
--add  package PowerShellStandard.Library \
--copy the code from src/library.fs \
--add script from scripts folder
```

There should be spaces before the ending slash.

Also, in Powershell `\` is represented by `backtick`